src/Controller/FigureController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\FigureType;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class FigureController extends AbstractController
+{
+    public function listTricks(EntityManagerInterface $entityManager)
+    {
+        return $entityManager->getRepository(Figure::class)->findAll();
```

Comment:
Pourquoi tu l'injecte pas directement le repository ? 

```suggestion
    public function listTricks(FigureRepository $figureRepository)
    {
        return $figureRepository->findAll();
```

Reply:
done